samples/namespace-monitoring/README.md

```diff
@@ -1,18 +1,18 @@
 # Namespace Monitoring
 
 This sample demonstrates how to customize a monitoring
-deployment to separate cluster monitoring from SAS Viya (namespace)
+deployment to separate cluster monitoring from SAS Viya platform (namespace)
 monitoring.
```

Comment:
This relevance of this sample isn't limited to deployments of "SAS Viya platform"  only; the same approach could be used to handle a similar need for any form of SAS Viya.